metpy/plots/skewt.py

```diff
@@ -770,10 +771,70 @@ def plot(self, u, v, **kwargs):
         u, v = delete_masked_points(u, v)
         return self.ax.plot(u, v, **line_args)
 
-    def plot_colormapped(self, u, v, c, **kwargs):
+    def plot_layers(self, u, v, hgt, bounds, colors, **kwargs):
```

Comment:
They've got slightly different functionalities. Plot_layers is designed specifically for layers in height coordinates and will interpolate to find the exact bounds, while the addition to plot_colormapped will color the hodograph by any arbitrary third variable without interpolating to the bounds. This difference is necessary because searchsorted doesn't work well with data that aren't increasing or decreasing monotonically, so adding the interpolated points when coloring the hodograph by temperature or wind speed doesn't work like it does with height.